### ISSUE TITLE
Possible to trigger validation outside of the form?

### ISSUE THREAD

**Issue by acsnorth** [running-coder/jquery-form-validation#32]
I'm trying to validate a form which is loaded into a modal popup via ajax. I've gotten the validator object to load after bringing the form into the DOM. My modal screen has action buttons in the footer, but these fall outside of the <form> element.

I would like to be able to trigger the validation via a JS call and not have a 'submit' button in the middle of the screen.

**Comment by acsnorth** [comment]
<img width="610" alt="modal validator" src="https://cloud.githubusercontent.com/assets/5615203/19692677/67754abe-9af6-11e6-8103-975199d2e5e1.png">

**Comment by acsnorth** [comment]
`<div class="modal-content">
    <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close" style="display: inline-block;">
            <span aria-hidden="true">×</span>
        </button>
        <h4 class="modal-title" id="myModalLabel">Newsletter Signup</h4>
    </div>
    <div class="modal-body"><form id="form-validation" name="form-validation" method="POST">
                    <div class="form-group">
                        <label class="form-label" for="validation-firstname">First name</label>
                        <input id="validation-firstname" class="form-control" name="validation[firstname]" data-validation="[NOTEMPTY]" data-validation-message="Please enter your first name." type="text">
                    </div>
                    <div class="form-group">
                        <label class="form-label" for="validation-lastname">Last name</label>
                        <input id="validation-lastname" class="form-control" name="validation[lastname]" data-validation="[NOTEMPTY]" data-validation-message="Please enter your last name." type="text">
                    </div>
                    <div class="form-group">
                        <label class="form-label" for="validation-email">Email</label>
                        <input id="validation-email" class="form-control" name="validation[email]" data-validation="[EMAIL]" data-validation-message="Please enter a valid email." type="text">
                    </div>
                    <div class="form-actions">
                        <button type="submit" class="btn btn-primary width-150">Validate</button>
                    </div>
                </form></div>
    <div class="modal-footer"><button type="button" class="btn" data-dismiss="modal">Close</button><button type="button" class="btn btn-primary" onclick="signUp();">Sign Up</button></div>
</div>`

**Comment by acsnorth** [comment]
Confessional debugging!!

I was able to achieve this by using the jQuery .trigger() API against the button.

`function signUp() {
	$("#valid_btn").trigger("click");
}`

**Comment by acsnorth** [comment]
<img width="572" alt="working" src="https://cloud.githubusercontent.com/assets/5615203/19693271/8c4c00e2-9af8-11e6-80ef-769a92c65811.png">
